DigitalLearningSolutions.Web/Styles/index.scss

```diff
@@ -199,8 +199,18 @@ ul.no-bullets {
     margin-bottom: 0;
 }
 
+@mixin hidden-submit-ie-fix {
+  // IE11 hack
+  @media screen and (-ms-high-contrast: active), (-ms-high-contrast: none) {
+    display: block;
+    position: absolute;
+    left: -100%;
+  }
+}
+
 .hidden-submit {
-  display: none;
+    display: none;
+    @include hidden-submit-ie-fix;

```

Comment:
Is this indent change actually accurate, or is it GitHub being funny?

Reply:
The indent change was indeed accurate. For some reason it had 4 spaces instead of the standard 2 we use for our scss files. I've changed it back to two.